modules/ROOT/pages/organization.adoc

```diff
@@ -19,47 +19,63 @@ An Anypoint Platform user who has been granted organization administrator privil
+== Organization Page
 
+In Access Management, click *Organization* in the left menu to view the Organization page. On the Organization page, you can:
+
+* View a hierarchical tree of all of the organizations you have permissions to view. You can view and edit properties of an organization by clicking the name.
+* Click the name of a group to view and edit information about it. What you can view and edit depends on your role. +
 [NOTE]
 Changing the name or domain name of an organization changes the deep links to any existing API Portals in your organization.
+* Add and delete business groups (if enabled).
+
 
 === Client ID and Client Secret
 
-Each environment in an organization has a unique client ID and a client secret used for authentication. To deploy proxies or APIs to CloudHub, you must use these values to configure a customer-hosted Mule Runtime or legacy API Gateway.
+Each environment and business group in an organization has a unique client ID and client secret that are used for authentication. You can use these to access assets within an organization if you are not an organization administrator. The client ID and password are generated by Anypoint Platform for each environment that you create, and they are globally unique.
+
+To deploy proxies or APIs to CloudHub, you must use these values to configure a customer-hosted Mule Runtime or legacy API Gateway.
 
 [NOTE]
 --
-Organization-level client IDs and client secrets are supported only for backward compatibility. In newer Anypoint Platform accounts, you must use the client ID and client secret for an _environment_. See xref:environments.adoc[Environments].
+Organization-level client IDs and client secrets are supported only for backward compatibility. In newer Anypoint Platform accounts, you must instead use the client ID and client secret for an _environment_. See xref:environments.adoc[Environments].
 --
 
-To view the client ID and secret for your organization:
+=== View the Client ID and Secret for Organizations and Environments
 
-. Sign into Anypoint Platform as an administrator.
-. Click *Organization*.
-. Click the name of the organization whose client ID/secret you want to view.
+. Sign into your Anypoint Platform account as a user with the Organization Administrator role.
+. Select the *Access Management* option from the main page, or select it from the left side bar by clicking the _menu_ button in the top left corner.
+. Click *Organization*, or *Environments* in the left menu.
+. Click the name of the organization or environment for which to view the client ID and secret.
 
 [NOTE]
 --
-This procedure does not apply to newer Anypoint Platform accounts, where the client ID and secret apply to environments rather than organizations.+
+Remember that for newer Anypoint Platform accounts, the client ID and secret apply to environments rather than organizations.
```

Comment:
Let's repeat the link to environment topics here, if the info is worth repeating, the link will be helpful?